### ISSUE TITLE
Compile errors

### ISSUE THREAD

**Issue by pulse00** [arranger1044/iSoul#8]
I'm trying to compile the project in Xcode 4.1 on Lion, but i'm getting errors already when cleaning the project:

```
    File /Users/username/Library/Developer/Xcode/DerivedData/iSoul-ervmuugfdxbpmohhmtpcemztfvyw/Build/Products/Release/libevent.framework depends on itself.  This target might include its own product.
    File /Users/username/Library/Developer/Xcode/DerivedData/iSoul-ervmuugfdxbpmohhmtpcemztfvyw/Build/Products/Release/libXMLFramework.framework depends on itself.  This target might include its own product.
```

I've cloned the repository, opened  `iSoul/iSoul.xcodeproj` in XCode and cleaned the project.

Are there any other initial steps before the project can be built?

**Comment by naos** [comment]
I'm having same problem. I'm trying to recompile it for 32-bit architecture in Xcode 4.0.2 on Snow Leopard, and I'm also getting same errors while cleaning the project.

**Comment by arranger1044** [comment]
Currently, I'm using XCode 3.6.2 to compile it, but I know that David was using XCode 4.x. We'd better wait for an answer from him.
Are you using release or build deployment targets?

**Comment by naos** [comment]
Error is reported while cleaning target museek (release configuration).

**Comment by naos** [comment]
Any news on this issue?
If there is still no progress, can you please compile it in Xcode 3.6.2 for 32-bit architecture and add the build to the repository.

**Comment by arranger1044** [comment]
I compiled a 32 bit version. You can find it under the Releases directory.

**Comment by naos** [comment]
Thanks!

**Comment by pulse00** [comment]
any ideas how to get rid of the build error?

**Comment by pulse00** [comment]
is there another way to link against libevent and libxml2 instead of embedding the 2 libs as xcode projects? maybe this will get rid of the errors.

**Comment by arranger1044** [comment]
Today I had a few minutes on a Lion machine and I can confirm the existence of the problem: it appears that under Xcode 4 libevent and libxml under the museek project 'depende on themselves' in the sense that they 'copy' themselves in the last pass of the build phase of museek (David could you please investigate more on this?).
So I deleted the clear dependency and made iSoule directly depend from libxml and libevent. Here you can the project http://www.mediafire.com/?ih8p90l94at9vxa . 
I'm not uploading to the repo since...it seems it gives compile errors under Xcode 3.6 this time : )

**Comment by pulse00** [comment]
thanks, that's great news. does this mean the project will either compile on xcode 3.x OR xcode 4, depending on the settings? 

i'll use the project you uploaded to mediafire in the meanwhile.

**Comment by pulse00** [comment]
I've already some changes i could contribute.

would it be an option for you to create a second branch with the xcode 4 changes? this way i could create the fork from the xcode4 branch and commit into the new branch until you upgraded the whole thing to xcode4.

**Comment by arranger1044** [comment]
I created the branch with the Xcode 4 settings. I apologize for being late but this has been a very full month and David seems missing too.

**Comment by pulse00** [comment]
thanks a lot - and no worries about the delay.

**Comment by djbe** [comment]
Whoops, never noticed this, I think I "unwatched" this repo by accident.
Anyway, yeah, noticed the error and got an easy fix for it I can commit. Gimme a few minutes

Edit: fixed in b006d52e86ddff344306fcb7969b5276f47c9980
